lab-kyler/lib/diskStorage.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+module.exports = exports = {};
+
+exports.storeItem = (schema, item) => {
+  if (!schema) return Promise.reject(new Error('no schema'));
+  if (!item || !item.id) return Promise.reject(new Error('no item or item ID'));
+
+  return fs.writeFileProm(`${__dirname}/../data/${schema}/${item.id}.json`, JSON.stringify(item))
+  .then( () => item)
+  .catch( err => Promise.reject(err));
+};
+
+exports.fetchItem = (schema, id) => {
+  if (!schema) return Promise.reject(new Error('no schema'));
+
+  return fs.readFileProm(`${__dirname}/../data/${schema}/${id}.json`)
+  .then (data => JSON.parse(data.toString()))
+  .catch (err => Promise.reject(err));
+};
+
+// exports.enumerate = (schema) => {
```

Comment:
Minor nitpick- always be sure to removed commented out code before pushing to the githubs.